library/Translate/test/Contextualise/Contextify.hs

```diff
@@ -36,39 +36,39 @@ sexpression =
 -- tests
 --------------------------------------------------------------------------------
 
-infixPlaceTest :: T.TestTree
-infixPlaceTest =
-  ( do
-      Right (ctx, _) <-
-        Contextualize.contextify (("Foo", desugared) :| [])
-      ctx Context.!? "+"
-        |> fmap ((\(Context.Def d) -> Context.defPrecedence d) . Context.extractValue)
-        |> (T.@=? Just (Context.Pred Context.Left 5))
-      |> T.testCase
-        "infix properly adds precedence"
-  )
-  where
-    Right desugared =
-      Desugar.op . AST.extractTopLevel
-        <$> Parser.parse "let (+) = 3 declare infixl (+) 5"
+-- infixPlaceTest :: T.TestTree
+-- infixPlaceTest =
+--   ( do
+--       Right (ctx, _) <-
+--         Contextualize.contextify (("Foo", desugared) :| [])
+--       ctx Context.!? "+"
+--         |> fmap ((\(Context.Def d) -> Context.defPrecedence d) . Context.extractValue)
+--         |> (T.@=? Just (Context.Pred Context.Left 5))
+--       |> T.testCase
+--         "infix properly adds precedence"
+--   )
+--   where
+--     Right desugared =
+--       Desugar.op . AST.extractTopLevel
+--         <$> Parser.parse "let (+) = 3 declare infixl (+) 5"
 
-sumConTest :: T.TestTree
-sumConTest =
-  T.testGroup
-    "Sum Constructors are properly added:"
-    [ T.testCase "Bool properly adds True" (test "True"),
-      T.testCase "Bool properly adds False" (test "False")
-    ]
-  where
-    test str = do
-      Right (ctx, _) <-
-        Contextualize.contextify (("Foo", desugared) :| [])
-      ctx Context.!? str
-        |> fmap Context.extractValue
-        |> (T.@=? Just (Context.SumCon (Context.Sum Nothing "bool")))
-    Right desugared =
-      Desugar.op . AST.extractTopLevel
-        <$> Parser.parse "type bool = True | False"
+-- sumConTest :: T.TestTree
+-- sumConTest =
+--   T.testGroup
+--     "Sum Constructors are properly added:"
+--     [ T.testCase "Bool properly adds True" (test "True"),
+--       T.testCase "Bool properly adds False" (test "False")
+--     ]
+--   where
+--     test str = do
+--       Right (ctx, _) <-
+--         Contextualize.contextify (("Foo", desugared) :| [])
+--       ctx Context.!? str
+--         |> fmap Context.extractValue
+--         |> (T.@=? Just (Context.SumCon (Context.Sum Nothing "bool")))
+--     Right desugared =
+--       Desugar.op . AST.extractTopLevel
+--         <$> Parser.parse "type bool = True | False"
```

Comment:
should these (and the imports above) be removed?

Reply:
Yes, just commented them out before I was getting ready to do mass deletion and moving things around